2-mandatory/1-create-functions.js

```diff
@@ -44,9 +77,37 @@ Write a function that:
 - Numbers greater 100 must be replaced with 100.
 */
 
-function formatPercentage() {
+function formatPercentage(arrayOfNumbers) {
+  const nothingAboveOneHundred = arrayOfNumbers.map((number) => {
+
+    if (number > 100) {
+      return 100;
+    } else {
+      return number;
+    }
+  });
+
+  const roundedNumbers = nothingAboveOneHundred.map((number) => {
+    //18.13 is an entry
+    //18.103 * 100 = 1810.3
+    //Math.round(1810.3) = 1810
+    //1810 / 100 = 18.1
+    return Math.round((number + Number.EPSILON) * 100) / 100;
+  });
+ 
+  const withPercentageSign = roundedNumbers.map((number) => {
+    return number + '%';
+  });
+
+  return withPercentageSign;
```

Comment:
the way you're breaking the logic down into steps with well named variables in great!